src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -46,9 +56,38 @@ private void handleCommandEntered() {
             commandTextField.setText("");
         } catch (CommandException | ParseException e) {
             setStyleToIndicateCommandFailure();
+        } finally {
```

Comment:
Great use of try-catch-finally here! 

Often, finally tend to be missed out. The use of finally improved the code readability.